script/check-st2-ok

```diff
@@ -24,9 +26,12 @@ retry_message() {
     echo "Don't worry! Occasionally something goes wrong, and you just have"
     echo "to kick the tires a bit. Try re-provisioning by executing the command:"
     echo ""
-    echo "${KICK_COMMAND}"
+    echo "DEBUG=1 ${KICK_COMMAND}"
     echo ""
     echo "If you don't happen to get the 'OK' after that, give us a shout!"
+    echo "Please also make sure to check and forward us the ${LOG_FILE} log file."
```

Comment:
It will read /var/log/puppet.log log file. Can you remove the word `log` and say, "share us the contents of ${LOG_FILE} either via pastebin or gist.github.com or any other pasting service of your choice."